src/Microsoft.ML.Data/DataLoadSave/Binary/BinaryLoader.cs

```diff
@@ -731,6 +731,12 @@ public void GetMetadata<TValue>(string kind, int col, ref TValue value)
         /// </summary>
         private const ulong ReaderVersion = MissingTextVersion;
```

Comment:
>ReaderVersion = MissingTextVersion [](start = 28, length = 34)

This was not updated, it should have been. We can read `StandardDataTypesVersion` so it should be that.